src/app/[locale]/layout.tsx

```diff
@@ -71,8 +72,11 @@ export default function Layout({ children, params }: PageProps & { children: Rea
     <html lang={locale}>
       <body className={fontClassName}>
         <OverrideGlobalLocalStorage />
-        {/* LATER-TASK add back once https://github.com/TheSGJ/nextjs-toploader/issues/66 is resolved */}
-        {/* <NextTopLoader /> */}
+        <NextTopLoader
+          color="#6200FF"
+          shadow="0 0 10px #6200FF,0 0 5px #6200FF"
```

Comment:
To avoid hard coding colors we could use a css variable injected by tailwind here
```suggestion
          color="hsl(var(--primary-cta))"
          shadow="0 0 10px hsl(var(--primary-cta)),0 0 5px hsl(var(--primary-cta))"
```

Reply:
Changed